src/app/components/layout/site-layout.tsx

```diff
@@ -0,0 +1,14 @@
+import Navbar from "@/app/components/primitives/navbar";
+import Footer from "@/app/components/sections/footer";
+
+export default function SiteLayout({ children }: { children: React.ReactNode }) {
```

Comment:
I think this should be renamed "LandingPageLayout" since it's only used in the landing page and FAQ page.

Reply:
fixed - please check.